### DISCUSSION TITLE
Which of the following features would lead to the greatest adoption of Mastodon

### DISCUSSION THREAD

**Discussion by dannyanders** [mastodon/mastodon#22017] [Polls]
Which of the following features would lead to the greatest adoption of Mastodon

**Comment by b-d-m-p** [comment]
People keep complaining about choosing a server. I was wondering suggesting a random server to sign up (with an option to choose another) for would make people feel it was smoother. You could have the servers opt in and limit it to a certain set of server permissions. Do all discussions of new features happen here?

**Reply by sfescape** [reply]
One of the biggest challenges the entire Fediverse faces is the fact you see different content depending on which server you join. Another challenge is the varying rules, stability, and longevity of instances.

Centralized solutions do not suffer from these issue. And selecting a random server means each new user will get a random first experience.

**Reply by ismailghedamsi** [reply]
The fact that Mastodon is decentralized makes specialized instances completely useless. For example, why would I join a tech instance if the feeds shows toots unrelated to tech? Lists of tags could solve the problem

**Comment by msmith9835** [comment]
I think an *opt-in* centralized (or replicated) directory allowing people to be found by name and profile details would help.

**Comment by shaedrich** [comment]
* It's important that instance owners stay able to pay for their servers. However, Mastodon is primarily free, so adding funding would still be used by a minority rather than a majority. So, while it might be nice to have, I doubt, it will have an impact on the adoption specifically
* Users usually don't provide their email address or phone number publicly (for good reasons), so I doubt, this would help discoverability a lot as is
* I don't think, Mastodon is and should necessarily be about big "influencers" how other social media platforms work. That's not a path most people in my experience want Mastodon to follow
* the age-old debate about mobile vs desktop. Please just let anyone use their Mastodon the way, they like and keep it at that. There is no need to favorize one group and therefore ditch the other

**Reply by dannyanders** [reply]
1. I think there are legitimate concerns about the sustainability of free servers. Something has to give.  Either server owners will eventually close their servers; leaving fewer and fewer options who's costs will increase exponentially.  You need to be able to fund them.  Donations will not be enough and advertising driven revenue is what leads to enshitification of most other platfroms.  A smalll subscription, even if it was volutnary, would help tremendously in makining it easier for members to support their instance.

2. The directory doesn't have to rely on phone or email. It could also leverage other social platforms; at least in an effort to ease the problem of finding people you are already connected to.  The lack of ability to find people you already are connected too elsewhere is definitely a problem for growth and engagement.  Of course, this could be done as opt-in so I don't think it would impact anyone who wanted to remain private.

3.Perhaps not.  But if you can't find your friends, and you can't find the people you DO want to hear from, then why are you here.  I would gladly follow many popular figures if they moved to mastodon.  But most have not and I have few reasons to come to mastodon daily.   Like it or not, public figures drive engagement.  

4. It's not a debate.  They are not mutually exclusive. You can use whatever you want. But mobile is clearlyy dominating when it comes to socially driven content feeds.

**Reply by shaedrich** [reply]
1. I completely agree with you. However, this is a separate issue. You asked what would lead to greater adoption. I don't think, it does.
2. Other social platforms sounds more interesting, and I think, opt-in is great, but opt-in also means, people need to be encouraged to do this, so that raises the question whether enough people will do that
3. \-
4. Mobile-first creates this debate in the first place. This _can_ turn into a form of enshittification. There's nothing wrong with creating a web application that works fine on all devices.

**Reply by dannyanders** [reply]
1. maybe you're right.  But I can't help wonder if those who run profitable instances would invest in marketing to draw more users.  Without a profit motive, marketing the benefits of Mastodon will continue to be slow.  Perhaps the mastodon community if fine with that.  But personally, I would use it far more if more people joined.  Right now it's still too limited to niche communities and early adopters.  I would love to have it as my primary feed, but it's not possible until there is mainstream adoption.  

2. I think this is a product question.  If you make it easy to do during account creation and explain the benefits, I think they would.  It's not unlike any other social  app that asks if you want to find your friends.  I think most people understand that apps provide more value if other people you know are on there.

4. I think there are some decent mobile clients now, so this might be a moot point.  I'm not sure if any are getting enought traction to lead to adoption though.

**Reply by shaedrich** [reply]
1. Using donations for self-less marketing of Mastodon, sounds like a good idea, assuming most instance admins have these marketing skills 👍🏻 
2. Sounds reasonable
